tailwind.config.js

```diff
@@ -388,6 +388,7 @@ module.exports = {
     zIndex: {
       0: 0,
       auto: 'auto',
+      navigation: 1,
```

Comment:
@mcibique not sure if we can remove this. You remember why we added it in the navigation?

Reply:
Talked with Milos this is resolved.